### PR TITLE
INC1502101 modkit install

### PR DESCRIPTION
**Pull request by gyear**
For INC150210 - `modkit-0.3.1rc1-GCCcore-12.2.0.eb`

* [ ] Assigned to reviewer

Default:
* [ ] EL8-icelake
* [ ] EL8-cascadelake

### CODE REVIEW

File: easyconfigs/m/modkit/modkit-0.3.1rc1-GCCcore-12.2.0-Rust-1.75.0.eb [hunk 1]
```diff
@@ -0,0 +1,627 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'Cargo'
+
+name = 'modkit'
+version = '0.3.1rc1'
+local_rustver = '1.75.0'
+versionsuffix = '-Rust-' + local_rustver
```
Comment (orbsmiv) [inline easyconfigs/m/modkit/modkit-0.3.1rc1-GCCcore-12.2.0-Rust-1.75.0.eb:7]: Looking at these easyconfs, I'm not sure it's necessary to have Rust in a version suffix, particularly as it's only a build dep:

- https://github.com/easybuilders/easybuild-easyconfigs/blob/cce1c8fdc5360d075d8c39cc34afb28b79fe40c4/easybuild/easyconfigs/m/mfqe/mfqe-0.5.0-GCC-12.3.0.eb
- https://github.com/easybuilders/easybuild-easyconfigs/blob/cce1c8fdc5360d075d8c39cc34afb28b79fe40c4/easybuild/easyconfigs/p/pydantic/pydantic-2.5.3-GCCcore-12.2.0.eb

**Review by orbsmiv — CHANGES_REQUESTED** [review]

File: easyconfigs/m/modkit/modkit-0.3.1rc1-GCCcore-12.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,627 @@
...
+    {'zerofrom-0.1.4.tar.gz': '91ec111ce797d0e0784a1116d0ddcdbea84322cd79e5d5ad173daeba4f93ab55'},
+    {'zerofrom-derive-0.1.4.tar.gz': '0ea7b4a3637ea8669cedf0f1fd5c286a17f3de97b8dd5a70a6c167a1730e63a5'},
+    {'zerovec-0.10.2.tar.gz': 'bb2cc8827d6c0994478a15c53f374f46fbd41bea663d809b14744bc42e6b109c'},
+    {'zerovec-derive-0.10.2.tar.gz': '97cf56601ee5052b4417d90c8755c6683473c926039908196cf35d99f893ebe7'},
+    {'modkit_fix_testing.patch': '02c860d65dad3da1943d824928cb517d5de0c04808e505fc50d5fae72bb19f44'},
+]
+
+builddependencies = [
+    ('binutils', '2.39'),
+    ('Rust', local_rustver),
```
Comment (orbsmiv) [inline easyconfigs/m/modkit/modkit-0.3.1rc1-GCCcore-12.2.0.eb:613]:
```suggestion
    ('Rust', '1.75.0'),
```

File: easyconfigs/m/modkit/modkit-0.3.1rc1-GCCcore-12.2.0.eb [hunk 3]
```diff
@@ -0,0 +1,627 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'Cargo'
+
+name = 'modkit'
+version = '0.3.1rc1'
+local_rustver = '1.75.0'
+versionsuffix = '-Rust-' + local_rustver
```
Comment (orbsmiv) [inline easyconfigs/m/modkit/modkit-0.3.1rc1-GCCcore-12.2.0.eb:7]:
```suggestion
```

File: easyconfigs/m/modkit/modkit-0.3.1rc1-GCCcore-12.2.0.eb [hunk 4]
```diff
@@ -0,0 +1,627 @@
...
+homepage = 'https://github.com/nanoporetech/modkit'
+description = """
+A bioinformatics tool for working with modified bases from Oxford Nanopore.
+Specifically for converting modBAM to bedMethyl files using best practices,
+but also manipulating modBAM files and generating summary statistics.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+crates = [
```
Comment (orbsmiv) [inline easyconfigs/m/modkit/modkit-0.3.1rc1-GCCcore-12.2.0.eb:18]: For consistency with other Cargo configs maybe move the `crates` list below the other deps?

**Review by orbsmiv — CHANGES_REQUESTED** [review]

**Comment by gyear** [comment]
User conda installed

**Pull request closed without merging**